.github/workflows/on-push-to-release-branch.yml

```diff
@@ -0,0 +1,63 @@
+name: On push to release branch
+
+on:
+  push:
+    branches: [ release ]
```

Comment:
this is fine for today, let's get it out there, but i just realized... we have been trying to switch things over to use the "release please" plugin, it would be really great to have another example of that. just something to keep in mind if you have time later this week.  @pgautier404 is a good contact if you have questions.